### PR TITLE
updates to schema module documenation

### PR DESCRIPTION
**Pull request by joshLong145**
updates to documentation from moving schemas all on chain
## Changes

- Item 1
- Item 2

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: docs/reference/modules/schemas.mdx [hunk 1]
```diff
@@ -8,29 +8,102 @@ import APIButton from "@site/src/components/APIButton";
...
+The Sonr Schema module is used to store the records of verifiable objects for a specific application powered by the Sonr Network. Schemas are used to create custom application protocols which can be asserted on in order to verify your application data. Application data uploaded can verify their data model through `Schemas`.
 
 ## Overview
 
 Schemas are implemented on the `IPLD Object Model` which allows developers to register specific application data schemas. See [IPLD Schema documentation](https://ipld.io/docs/schemas)
 
 ## Concepts
 
-### Schema Kind Definition
+### What Is records
```
Comment (mcjcloud) [inline docs/reference/modules/schemas.mdx:19]: Should be "WhatIs"

File: docs/reference/modules/schemas.mdx [hunk 2]
```diff
@@ -8,29 +8,102 @@ import APIButton from "@site/src/components/APIButton";
...
+
+
+Currently a schema cannot be modified once it has been created. To Modify A Schema, create a new Schema.
+
+----
 
 ### Schema Definition
 
-A `Schema Definition` is used to describe an application Schema that will be stored for later assertion against. The provided `Schema Definition` is then used to Derive both the `WhatIs` and `Schema Reference` that will be recorded on chain. Schemas comply to the `IPLD Object` specification.
+A `Schema Definition` is used to describe an application Schema that will be stored for later assertion against. The provided `Schema Definition` is then used to Derive both the `WhatIs` that will be recorded on chain. Schemas comply to the `IPLD Object` specification.
```
Comment (mcjcloud) [inline docs/reference/modules/schemas.mdx:93]: 1. `Schema Definition` -> `SchemaDefinition`
2. Derive -> derive
3. delete "both" from "derive both the"

File: docs/reference/modules/schemas.mdx [hunk 3]
```diff
@@ -39,6 +112,10 @@ message SchemaDefinition{
 
 ---
 
+### Schema Kind Definition
+
+Schema's declare the intended type of a property though IPLD Type `KIND` Each `KIND` is mapped to the name of a property. Each property defined must map to one of the given `IPLD` `KINDS`
```
Comment (mcjcloud) [inline docs/reference/modules/schemas.mdx:117]: Add period: "...IPLD Type `KIND`(.) Each `KIND` is mapped..."

Comment (mcjcloud) [reply docs/reference/modules/schemas.mdx:117]: Change "`IPLD` `KINDS`" to "`IPLD KINDS`"

**Review by mcjcloud — CHANGES_REQUESTED** [review]

**Review by ma-sonr — APPROVED** [review]

**Pull request merged**